Assets/MixedRealityToolkit/Sharing/Scripts/SyncSettings.cs

```diff
@@ -100,7 +99,7 @@ public void Initialize()
             }
         }
 
-        private static Assembly[] GetAssemblies()
+        private static System.Reflection.Assembly[] GetAssemblies()
```

Comment:
This fixes namespace issues by moving it inside the class instead of declaring it above.

Reply:
If this fixes the namespace issue, do we still need to move to 4.6?

Reply:
Also, if we do move to 4.6, this only *sorta* fixes the namespace issue. With this change, it's still using the Toolkit's old extension method instead of the new official one. Should we not move to the platform's extension method and delete the Toolkit's, since the old Toolkit one only existed to fill the gap of an official one?

Reply:
That's a good point, I could remove the them